Monika After Story/game/event-handler.rpy

```diff
@@ -2032,6 +2032,13 @@ label prompt_menu:
         if cb_label is not None:
             call expression cb_label
 
+        # NOTE: we only need to enable music hotkey since we are in dlg mode
+        #$ mas_DropShield_idle()
+        $ store.mas_hotkeys.music_enabled = True
```

Comment:
why was this moved here, this should be last to prevent interruptions in flow

Reply:
Yeah, I moved it as during testing it took significantly longer to change the shields.

Reply:
Actually, time difference wasn't noticeable between the two, will swap back